lib/stacked_card_carousel.dart

```diff
@@ -54,6 +63,8 @@ class StackedCardCarousel extends StatefulWidget {
   final bool _applyTextScaleFactor;
   final PageController _pageController;
   final OnPageChanged? _onPageChanged;
+  final Axis scrollDirection;
```

Comment:
💡 We should follow the convention here and use private variables. Please check the other variables as example.